### DISCUSSION TITLE
Fresh installation issue

### DISCUSSION THREAD

**Discussion by Patberissy** [Koenkk/zigbee2mqtt#5790] [General]
Hi all.
I am new to this community and trying to solve something looking like a very basic issue.

After a full fresh Debian Buster installation on Raspberry PI3B, I am trying to install zigbee2mqtt from scratch and fail anytime.
I apply exactly the procedure described in "https://www.zigbee2mqtt.io/getting_started/running_zigbee2mqtt.html".
Failing permanently at the same place when the installation is launched : See herafter for a copy of the console :

pi@raspberrypi:/opt/zigbee2mqtt $ npm ci
npm ERR! code ENOENT
npm ERR! syscall open
npm ERR! path /opt/zigbee2mqtt/node_modules/zigbee-herdsman-converters/node_modules/agent-base/package.json
npm ERR! errno -2
npm ERR! enoent ENOENT: no such file or directory, open '/opt/zigbee2mqtt/node_modules/zigbee-herdsman-converters/node_modules/agent-base/package.json'
npm ERR! enoent This is related to npm not being able to find a file.
npm ERR! enoent 

npm ERR! A complete log of this run can be found in:
npm ERR!     /home/pi/.npm/_logs/2021-01-17T09_38_19_609Z-debug.log

[2021-01-17T09_38_19_609Z-debug.log](https://github.com/Koenkk/zigbee2mqtt/files/5825924/2021-01-17T09_38_19_609Z-debug.log)
Ideas for solution ?
Thanks for your help.

**Comment by Patberissy** [comment]
Hello.
Good news : problem solved. 
Solution : the procedure "https://www.zigbee2mqtt.io/getting_started/running_zigbee2mqtt.html" suggests to install node.js  version 12.x. Problem solved through installation of node.js 15.x instead (replace "setup_12.x" by "setup_15.x" in the installing procedure).
Regards,

**Reply by m0wlheld** [reply]
Go for the [Docker](https://www.zigbee2mqtt.io/information/docker.html) install and leave all the dependency hell of npm, python or php behind...
